lib/error_tracker.ex

```diff
@@ -107,6 +107,11 @@ defmodule ErrorTracker do
   * A `{kind, exception}` tuple in which case the information is converted to
   an Elixir exception (if possible) and stored.
   """
+
+  @type exception :: struct() | {atom(), struct()}
```

Comment:
This spec is incorrect. Not every `struct` is valid, only exception struct. For tuples we should take a look at `Exception.normalize`.